docs/index.md

```diff
@@ -15,9 +15,9 @@ description: |-
 
 ### Required
```

Comment:
@DomWeldon The updates to the index.md will actually need to be done via the cloudcraft/provider.go file.
You will need to make the changes like so.
+++ b/cloudcraft/provider.go
@@ -14,20 +14,20 @@ func Provider() *schema.Provider {
                                Type:        schema.TypeString,
                                Required:    true,
                                DefaultFunc: schema.EnvDefaultFunc("CLOUDCRAFT_APITOKEN", nil),
-                               Description: "apikey for cloudcraft",
+                               Description: "apikey for cloudcraft, can be set using environment variable CLOUDCRAFT_APITOKEN",
                                Sensitive:   true,
                        },
                        "baseurl": {
                                Type:        schema.TypeString,
                                Optional:    true,
                                DefaultFunc: schema.EnvDefaultFunc("CLOUDCRAFT_HOST", "api.cloudcraft.co"),
-                               Description: "Host URL for cloudcraft",
+                               Description: "Host URL for cloudcraft, can be set using environment variable CLOUDCRAFT_HOST",
                        },
                        "max_retries": {
                                Type:        schema.TypeInt,
                                Optional:    true,
                                DefaultFunc: schema.EnvDefaultFunc("CLOUDCRAFT_MAX_RETRIES", 1),
-                               Description: "Max Retries",
+                               Description: "Max Retries, can be set using environment variable CLOUDCRAFT_MAX_RETRIES",



Once you have done these changes, run the ./tfplugindocs to generate the documentation again, this will create the docs folder with the updated comments made to the provider.